### ISSUE TITLE
How do you finetune 32k context length ?

### ISSUE THREAD

**Issue by thusinh1969** [QwenLM/Qwen2-VL#232]
Just out of curiosity. I have been trying to do likes 30 images 448x to maximize of 16k context length with all optmizer, gradient checkpointing with LoRA on **H100 NVL 94G** and that is it all I can do.

Any hint would be appreciated.
Steve
